data/json/regional_map_settings.json

```diff
@@ -916,7 +916,51 @@
       "base_acid": 0.0,
       "base_wind": 3.4,
       "base_wind_distrib_peaks": 80,
-      "base_wind_season_variation": 50
+      "base_wind_season_variation": 50,
+      "mist": {
```

Comment:
This should be generalized to an array of localized weather objects with jsonized properties and effects.